.github/actions/install-build-tools/action.yml

```diff
@@ -6,6 +6,15 @@ runs:
   - name: Install Build Tools
     shell: bash
     run: |
-      export BUILD_TOOLS_SHA=eeb1a11392e4cec08fd926c93b31ab556dc0c23b
+      if [ "$(expr substr $(uname -s) 1 10)" == "MSYS_NT-10" ]; then
+        git config --global core.filemode false
```

Comment:
Doesn't build-tools itself [set this](https://github.com/electron/build-tools/blob/07ad97f145f9fb82aee17d76eb38669a6c035122/src/utils/git.js#L11-L37)?